src/components/panels/ConfigPanel/_config-panel.scss

```diff
@@ -5,8 +5,14 @@
   $magic-height: 200px;
   $magic-min-height: calc(100vh - #{$magic-height});
 
-  @media (max-width: $breakpoint-large) {
-    grid-template-columns: 1fr !important;
+  .config-content {
+    @media (max-width: $breakpoint-large) {
+      grid-template-columns: 1fr !important;
+    }
+
+    &.row {
```

Comment:
You removed the `row` class right?

Reply:
It is easy to miss but I removed it from the wrapper and into the `FadeIn` component so this is still required....I'm refactoring how this is all laid out in a follow-up to better work with the confirmation dialog so this will change a bit anyways very soon.